### PR TITLE
{bio}[gompi/2022b] ProBiS v20230403

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ItIsI-Orient** [comment]
@boegelbot please test @ generoso

**Comment by ItIsI-Orient** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/p/ProBiS/ProBiS-20230403-gompi-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+source_urls = ['https://gitlab.com/janezkonc/probis/-/archive/master/']
+sources = ['%s.tar.gz' % local_commit]
+checksums = ['5f81df7498daefe0e66082561f31d4c66f75cf1eebbed6430e3d78dd3097896b']
+
+dependencies = [
+    ('GSL', '2.7'),
+]
+
+buildopts = 'probis CC="mpic++" CFLAGS="-std=c++11 -O3 -Wall -Wno-unused-function -DNDEBUG -pthread"'
```
Comment (boegel) [inline easybuild/easyconfigs/p/ProBiS/ProBiS-20230403-gompi-2022b.eb:21]: avoid hardcoding `mpic++` and `-std=c++11 -O3`:

```suggestion
buildopts = 'probis CC="$MPICXX" CFLAGS="$CFLAGS -Wall -Wno-unused-function -DNDEBUG -pthread"'
```

File: easybuild/easyconfigs/p/ProBiS/ProBiS-20230403-gompi-2022b.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+name = 'ProBiS'
+version = '20230403'
+local_commit = 'f10a9f4c'
+
+homepage = 'http://insilab.org/probis-algorithm'
+description = """ProBiS algorithm aligns and superimposes complete protein surfaces,
+surface motifs, or protein binding sites."""
+
+toolchain = {'name': 'gompi', 'version': '2022b'}
```
Comment (boegel) [inline easybuild/easyconfigs/p/ProBiS/ProBiS-20230403-gompi-2022b.eb:11]:
```suggestion
toolchain = {'name': 'gompi', 'version': '2022b'}
toolchainopts = {'cstd': 'c++11'}

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3157.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/bba9cbbbd63f1d2d6160d99a0aed095e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**